library/oneview_enclosure_facts.py

```diff
@@ -55,23 +62,72 @@
   oneview_enclosure_facts:
     config: "{{ config_file_path }}"
     name: "Enclosure-Name"
+  delegate_to: localhost
+
+- debug: var=enclosures
+
+- name: Gather facts about an Enclosure by name with options
+  oneview_enclosure_facts:
+    config: "{{ config_file_path }}"
+    name: 'Test-Enclosure'
+    options:
+      - script                       # optional
+      - environmentalConfiguration   # optional
+      - utilization                  # optional
+  delegate_to: localhost
+
+- debug: var=enclosures
+- debug: var=enclosure_script
+- debug: var=enclosure_environmental_configuration
+- debug: var=enclosure_utilization
+
+- name: "Gather facts about an Enclosure with temperature data at a resolution of one sample per day, between two
+         specified dates"
+  oneview_enclosure_facts:
+    config: "{{ config_file_path }}"
+    name: 'Test-Enclosure'
+    options:
+      - utilization                  # optional
+          fields: 'AmbientTemperature'
+          startDate: '2016-07-01T14:29:42.000Z'
+          endDate: '2018-07-01T03:29:42.000Z'
+          view: 'day'
+          refresh: False
+  delegate_to: localhost
 
 - debug: var=enclosures
+- debug: var=enclosure_environmental_configuration
```

Comment:
I think that the  returned `var` is not `enclosure_environmental_configuration` but  `enclosure_utilization`